service/resource/certificate/desired.go

```diff
@@ -30,28 +30,36 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 
 	r.logger.Log("debug", "fetching certificates")
 	certificateFiles := []certificateFile{}
+
 	for _, service := range validServices {
-		namespace, name := prometheus.GetCertificateName(service)
+		clusterID := prometheus.GetClusterID(service)
```

Comment:
This is an implementation that works for now. Will be replaced with certificatetpr implementation shortly.